scripts/build_static

```diff
@@ -26,8 +26,19 @@ export external_endpoint="https://ssg-${environment}.vote.gov"
 cd ${app_path}
 echo "Running 'drush tome:static' in '${environment}'..."
 drush tome:static --uri=${external_endpoint} --process-count=2 --retry-count=0 -y
+drush tome:static-export-path '/sitemap.xml,/sitemap_generator/default/sitemap.xsl' --uri=${external_endpoint} --process-count=2 --retry-count=0 -y
 echo "'drush tome:static' task completed!"
 
+echo "Adding missing Core assets vendor directory"
+mkdir -p ${html_path}/core/assets
+cp -rfp ${app_path}/web/core/assets/vendor ${html_path}/core/assets/
+echo "Missing Core assets files completed!"
```

Comment:
@daniel-crowder can you give me the details on why this is needed for the sitemap?

Reply:
The only thing that it really needs is the jquery and any other js library to support the column sorting function that's available on the sitemap.xml page. I included the entire directory as a catch all, but plan on revisiting once we have the entire site flushed out and built via Tome to remove anything that we absolutely don't need like the ckeditor files.